[5.11.0] Update sections with callback regex configuration (#3715)
wso2/docs-is: en/docs/learn/password-recovery.md

.

    ![notification-based-password-recovery](../assets/img/using-wso2-identity-server/notification-based-password-recovery-conf.png)

    !!! note

>         - The recommended  **Recovery callback URL regex** to use when testing the product is `^https:\/\/localhost:9443\/.*`. But user should modify it meet their requirements when they are deploying the product.

Reviewer: ```suggestion
        - The recommended  **Recovery callback URL regex** to use when testing the product is `^https:\/\/localhost:9443\/.*`. However, users should modify it to meet their requirements when they deploy the product.
```
